guides/index.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Guides overview
+keywords: guides, overview
+title: Guides overview
+notoc: true
+---
+
+This section includes guides where you can learn how to set up your Docker environment and start containerizing your applications.
+
+<div class="component-container">
```

Comment:
Thanks @dvdksn, what do you think about keeping the card titles the same as the nav sections, but then adding more complimentary info and links in the descriptions?

For example, have the "Language-specific guides" card keep the same title, but update its description to expose and link to each language. 

The reason for keeping the the card titles the same as the left nav is so that it's easy to relate the overview structure with the nav structure. Another alternative is to add the nav sections as titles on the overview page, and then create cards for the main topics under each title. The only sticky point is that there are some standalone top-level topics that will probably need to be styled differently because there are no sub-topics for cards.